src/main/resources/processes/BF2/BF2.bpmn

```diff
@@ -230,6 +230,7 @@
         <camunda:out source="ReturnCase" target="ReturnCase" />
         <camunda:in sourceExpression="${execution.processBusinessKey}" target="CaseUUID" />
         <camunda:in source="SeriousMisconduct" target="CompType" />
+        <camunda:in source="Service" target="CompType" />
```

Comment:
How does this work with the line above? 232 sets CompType to SeriousMisconduct and then this sets to Service. This would override L232?